README.md

```diff
@@ -1,177 +1,204 @@
 # OPRE OPS
-[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit) [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-This is a prototpye of OPRE's research Portfolio management System, or **OPS**. The finished product will replace OPRE's prior system, MAPS, which [is archived here](https://github.com/HHS/MAPS-app). The purpose of OPS can be found on [the wiki](https://github.com/HHS/OPRE-OPS/wiki).
+This is the OPRE's Research Portfolio Management System, or OPS. The finished product will replace OPRE's prior system,
+[MAPS](https://github.com/HHS/MAPS-app). The purpose of OPS can be found on
+[the wiki](https://github.com/HHS/OPRE-OPS/wiki).
 
-Technologies used in this prototype:
-* Python
-* Django 4.x
-* React
-* PostgreSQL
-* Cloud.gov
+## Dependencies
 
-## Getting Started
+At a bare minimum, you need [Docker](https://www.docker.com) installed to run the application locally.  If you want to do
```

Comment:
I don't recall, but does `Docker` include `Compose` now? I know it didn't in the past, and it was a separate install on some systems.

Reply:
Oh, good point.  I know it does with Docker Desktop for ____.  Let me check.

Reply:
I added a bit that we also require Docker Compose.  The webpage linked to mentions that you might already have it through Docker Desktop and also includes instructions on how to install it if you just have the engine/CLI installed.

Reply:
👍